docs/Architecture.md

```diff
@@ -81,10 +80,10 @@ It contains code for starting all the _benchmark runners_ and is aware of all ex
 
 _Global runner_ is started by the user. User may pass some or all of the variables listed in the [methodology](Methodology.md) as command line arguments.
 
-__The exact specifications are to be developed along with the _global runner_ itself__.
+For the complete documentation refer to [GlobalRunner.md](./GlobalRunner.md).
 
 ### Result-Processing Scripts
 
-Two scripts. One looks at the output files and compares the results in them to the manually computed correct ones, the other produces visualizations of the timings.
-
-__The exact specifications are to be developed along with the _result-processing scripts_ themselves__.
+Scripts that consume the outputs of the _benchmark runners_ and process them.
+Currently we have the following scripts:
+- [ADBench/plot_graphs.py](docs/PlotCreating.md)
```

Comment:
Seems like a broken link. As far as this file in the `/docs` directory, the link should be `./PlotCreating.md`, I think

Reply:
Fixed.